gapil/format/format.go

```diff
@@ -162,6 +162,8 @@ func (p *printer) markup(n ast.Node) {
 			p.inject(n.Fields[0], beforePrefix, "•»")
 			p.inject(n.Fields[c-1], beforeSuffix, "«•")
 		}
+	case *ast.Clear:
+		//p.inject(n.Value, afterPrefix, "•")
```

Comment:
@ben-clayton This I was not sure about, I can't quite wrap my head around how the formatter works.